cleanlab_studio/studio/enrichment.py

```diff
@@ -17,6 +17,7 @@
 import pandas as pd
```

Comment:
please update the PR description with:

1. User code if they just want to do some real-time data enrichment quickly.

2. User code if they want to first run data enrichment project over a big static dataset, and then later want to run some real-time data enrichment over additional data.